realm/realm-library/src/main/java/io/realm/internal/ColumnInfo.java

```diff
@@ -217,39 +222,28 @@ public String toString() {
      * <p>
      * No validation done here.  Presuming that all necessary validation takes place in {@code Proxy.validateTable}.
      *
-     * @param table The table to search for the column.
      * @param columnName The name of the column whose index is sought.
-     * @param columnType Type RealmType of the column.
+     * @param objectSchemaInfo the {@link OsObjectSchemaInfo} for the corresponding {@code RealmObject}.
      * @return the index of the column in the table
      */
-    @SuppressWarnings("unused")
```

Comment:
Isn't it needed anymore? It looks like invoking `addColumnDetails` is similar with the previous code.